### ISSUE TITLE
error at running the app at new dir

### ISSUE THREAD

**Issue by assafSabag** [startach/haprev#4]
output of npm ls

npm ERR! peer dep missing: eslint@^3.17.0, required by eslint-plugin-react-native@2.3.2
npm ERR! peer dep missing: eslint-plugin-jsx-a11y@^5.1.1, required by eslint-config-airbnb@15.1.0
npm ERR! peer dep missing: react-native@>=0.46.0, required by native-base@2.3.1
npm ERR! peer dep missing: react@>=15.3.1, required by lottie-react-native@1.1.1
npm ERR! peer dep missing: react@>=15.4.0, required by react-native-branch@2.0.0-beta.3
npm ERR! peer dep missing: react@>=15.4.0, required by react-native-maps@0.14.0

**Comment by eladcandroid** [comment]
You can choose one of the following solutions:

**Solution 1:**
Remove node_modules directory
Update your npm version 
(by this explanation: https://stackoverflow.com/questions/23393707/how-to-update-npm)
Run "npm cache clean"
Reopen console
Try to run "npm install" again

**Solution 2:**
Remove node_modules directory
Install yarn (https://yarnpkg.com/en/)
Reopen console
Run "yarn" instead of "npm install" and tell me if you get any error
